src/plugins/slowed/index.tsx

```
@@ -0,0 +1,140 @@
+
+      const getVideo = () => document.querySelector<HTMLVideoElement>('video');
+
+      document.getElementById('sr-panel')?.remove();
+
+      const panel = document.createElement('div');
+      panel.id = 'sr-panel';
+      document.body.appendChild(panel);
+
+      const dispose = render(() => (
+        <div class="sr-container">
+          <div class="sr-header" onClick={() => setCollapsed(!collapsed())} style="cursor: pointer;">
```

ESLint

[error] Replace `<div·class="sr-header"·onClick={()·=>·setCollapsed(!collapsed())}·style="cursor:·pointer;"` with `··<div⏎··············class="sr-header"⏎··············onClick={()·=>·setCollapsed(!collapsed())}⏎··············style="cursor:·pointer;"⏎············`

(prettier/prettier)